### PR TITLE
Update session id logic for dtls13

### PR DESCRIPTION
**Pull request by fwh-dc**

### CODE REVIEW

**Comment by fwh-dc** [comment]
Ci failure looks irrelevant

File: ssl/ssl_txt.c [hunk 1]
```diff
@@ -34,10 +34,12 @@ int SSL_SESSION_print(BIO *bp, const SSL_SESSION *x)
     size_t i;
     const char *s;
     int istls13;
+    int isdtls13;
```
Comment (t8m) [inline ssl/ssl_txt.c:37]: There is no point in adding a variable if you actually need isversion13 variable.

**Review by t8m — CHANGES_REQUESTED** [review]

File: ssl/ssl_txt.c (hunk 1 above)
Comment (fwh-dc) [reply ssl/ssl_txt.c:37]: Agree!

**Review by mattcaswell — APPROVED** [review]

**Review by t8m — APPROVED** [review]

**Comment by mattcaswell** [comment]
Pushed.

**Pull request closed without merging**